lib/bolt/outputter/human.rb

```diff
@@ -445,6 +482,8 @@ def print_plan_result(plan_result)
           @stream.puts("Plan completed successfully with no result")
         when Bolt::ApplyFailure, Bolt::RunFailure
           print_result_set(value.result_set)
+        when Bolt::ContainerResult
```

Comment:
There should be a case for handling `Bolt::ContainerFailure` errors as well.

Reply:
Hm, I actually do think so here at least - we don't have special handling for similar errors like `ParallelizeFailure` or plan evaluation errors, and I think that handling is meant for failures with result sets. It should print a nicer error when the `container_finish` event is passed though, which it's not doing...

Reply:
Ok, ended up running away with this a bit :sweat_smile: but there should be pretty output for all of these instances now (ContainerFailure, results with verbose output, plan results, regular results, etc.)